src/styles/index.ts

```diff
@@ -4725,6 +4724,13 @@ const styles = (theme: ThemeColors) =>
             lineHeight: variables.lineHeightXXLarge,
         },
 
+        videoPlayerPreview: {
+            width: '100%',
+            height: '100%',
+            borderRadius: variables.componentBorderRadiusNormal,
+            backgroundColor: theme.highlightBG,
```

Comment:
Coming from this issue https://github.com/Expensify/App/issues/58130, adding this `backgroundColor` here causes a mismatch with the app’s background color. More context here: https://github.com/Expensify/App/issues/58130#issuecomment-2712146467